### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by HammyHavoc** [erxes/erxes#1941]
**Is your enhancement request related to a problem? Please describe.**
My colleague is legally blind and cannot use UX that features bright white. Dark mode is required for accessibility friendliness.

**Describe the solution you'd like**
Add a dark mode.
